layout.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please add IDE-specific files to `.gitignore`.

Reply:
Should be ok now I think.